__main__.py

```diff
@@ -1,2 +1,25 @@
+import argparse
+
 if __name__ == "__main__":
-    print("oh, what up?")
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--api", help="Get data from API", type=bool,
+                        default=False, action=argparse.BooleanOptionalAction)
+    parser.add_argument("--parse-data",
+                        help="Parse data from the API and external sources",
+                        type=bool, default=False,
+                        action=argparse.BooleanOptionalAction)
+    parser.add_argument("--visualize", help="Use data from the ",
+                        type=bool, default=False,
+                        action=argparse.BooleanOptionalAction)
+
+    args = parser.parse_args()
+
+    if args.api:
+        print("We are gonna do some API work")
+
+    if args.parse_data:
+        print("We are gonna do some data parsing work")
+
+    if args.visualize:
+        print("We are gonna do some visualizations")
```

Comment:
These flags don't actually do anything, aside from call those print statements, but they do make it so that when those functions are in place it just requires the addition of an import line and the function call to the `__main.__.py` file to see have your functionality backed in the program. The premise is that all of the setup is done and people can sort of plug-and-play as they start to create modules/functions.

Reply:
Yeah that's the vibe I was picking up so I'm glad to have it confirmed.